src/settings/components/Capabilities/helpers/index.js

```diff
@@ -4,7 +4,7 @@ import { columnTranslations } from '../../../../constants/translations';
 export const useCheckboxAriaStates = () => {
   const { formatMessage } = useIntl();
 
-  const getCheckBoxAriaLabel = (action, resource) => `${formatMessage(columnTranslations[action])} ${resource}`;
```

Comment:
Aside: Yikes, `constants/translations` really makes me nervous because now translation keys are copied across multiple locations: `src/constants/translations` and `translations/ui-authorization-roles/*.json`. While I understand the theoretical value of `description`, I'm not sure it's ever actually made available to the translator since this file is never sent to Lokalise. 

We need to investigate this pattern and decide if we want to embrace it (and implement it everywhere) or reject it (and remove it everywhere).